src/Octoshift/Handlers/WaitForMigrationCommandHandler.cs

```diff
@@ -108,20 +108,22 @@ private async Task WaitForRepositoryMigration(string migrationId, string githubP
             if (RepositoryMigrationStatus.IsSucceeded(state))
             {
                 _log.LogSuccess($"Migration {migrationId} succeeded for {repositoryName}");
+                _log.LogInformation($"Migration log available at {migrationLogUrl} or by running `gh gei download-logs`");
```

Comment:
Wasn't there a discussion about making the `gei` bit dynamic so it shows the proper RootCommand (gei/bbs2gh/ado2gh)?

Reply:
I think we should do that to make sure that the command works consistently.

Reply:
Ah shoot- that was this bit, right? `($"Migration log available at {url} or by running `gh {CliContext.RootCommand} download-logs`");`

I think that got lost in the noise at some point (or overwritten), I'll update now.

Reply:
Weird. This is already updated locally and it's telling me these changes are already included in the source (after a fetch and pull)... So, it's gonna be that kind of day today.

Reply:
Huh. TIL that `{CLIContext.RootCommand}` gets changed to `gei` on push...

I updated with a backtick on my last push. the backtick was uploaded but `gei` is still being shown instead of the rootcommand... This is what I see locally:

![D68A4C49-C387-4733-8A69-B4B7D4EDEBF8_4_5005_c](https://user-images.githubusercontent.com/38845272/225732168-a8c96598-177f-474b-bbb8-31cf04af299c.jpeg)

Reply:
Well, apparently I don't either! I just switched back to this branch again and now it's not showing the `CliContext.RootCommand` at all 😕 Made the changes and just pushed up though...

 Maybe I am just crazy? Entirely plausible at this point 🙄

Reply:
I have no clue why that was being so weird. Fixed now though-- all's well that ends well? 🤔